### PR TITLE
GH-4757: Copyedit /front-end-dev-guide/theme-best-practice.md page

### LINKED ISSUE #4757: Feedback on page: /guides/v2.3/frontend-dev-guide/theme-best-practice.html
# General issue

Objective grammar errors.

## Description:

The punctuation and grammar are poor.

Poor grammar.

For example:

"Keep in mind, that after Magento instances, updates or upgrades something can change in default templates, layouts, and styles. So it is recommended to check if the changes effected the files overridden in your theme and copy changes if any, to your templates, layouts, and styles."

This contains a comma splice, a possibly misplaced word, an outright incorrect word, and a poorly designed sentence fragment.

## Possible solutions:

For example:
"Keep in mind that Magento instances, updates, or upgrades can change default templates, layouts, and styles, so it is recommended to check if such changes affect the files overridden in your theme and to copy any changes into your overrides."

## Additional information:

I'm a line editor with over a decade of experience.

### PR DESCRIPTION
**Pull request by shrielenee**
## Purpose of this pull request

Copy edit the /front-end-dev-guide/theme-best-practice.md page by request from GitHub issue #4757 

Preview build: https://devdocs.magedevteam.com/784/guides/v2.3/frontend-dev-guide/theme-best-practice.html

Fixes #4757 

This pull request (PR) copyedits the /front-end-dev-guide/theme-best-practice.md page by request from GitHub issue #4757

## Affected DevDocs pages

- https://devdocs.magento.com/guides/v2.3/frontend-dev-guide/theme-best-practice.html
- https://devdocs.magento.com/guides/v2.2/frontend-dev-guide/theme-best-practice.html

## Links to Magento source code

- ...

### CODE REVIEW

File: guides/v2.2/frontend-dev-guide/theme-best-practice.md [hunk 1]
```diff
@@ -7,26 +7,22 @@ functional_areas:
...
+1. If you need to change the wording in the user interface, [add custom CSV dictionary files]({{ page.baseurl }}/frontend-dev-guide/translations/theme_dictionary.html) instead of overriding `.phtml` templates.
+1. Use [the CSS critical path]({{ page.baseurl }}/frontend-dev-guide/css-topics/css-critical-path.html) to render the page much faster.
 
-Keep in mind that, after updating or upgrading Magento instances, there may be changes to default templates, layouts, and styles. So it is recommended to check if those changes affect any files overridden in your theme and if so, copy the changes to your templates, layouts, and styles.
+After updating or upgrading Magento instances, there may be changes to default templates, layouts, and styles. It is recommended that you check if those changes affect any files overridden in your theme, and if so, copy the changes to your templates, layouts, and styles.
```
Comment (hickskenh) [inline guides/v2.2/frontend-dev-guide/theme-best-practice.md]: suggest rewording to bring the action closer to the beginning:
After updating or upgrading Magento instances, check for changes in any files that are overridden by your theme. If there were changes to default templates, layouts, or styles, copy those changes to your templates, layouts, and styles.

File: guides/v2.2/frontend-dev-guide/theme-best-practice.md [hunk 2]
```diff
@@ -7,26 +7,22 @@ functional_areas:
   - Standards
 ---
 
-This topic describes the recommended approaches and things to consider when working on a custom theme.
+Utilizing theme development best practices give you a better chance of avoiding conflicts and issues with your theme after you update or upgrade your Magento instance or install a custom extension.
```
Comment (hickskenh) [inline guides/v2.2/frontend-dev-guide/theme-best-practice.md]: maybe add "customized" before theme.  

also, i see this a lot, but aren't update and upgrade the same thing?

File: guides/v2.2/frontend-dev-guide/theme-best-practice.md [hunk 3]
```diff
@@ -7,26 +7,22 @@ functional_areas:
...
 
-These approaches give higher chances to avoid conflicts and issues with your theme after your Magento instance get updated or upgraded, or a custom extension is installed.
-
-## Theme development best practice
+We recommend using the following best practices when developing themes:
```
Comment (hickskenh) [inline guides/v2.2/frontend-dev-guide/theme-best-practice.md]: maybe add "custom" before themes?

File: guides/v2.2/frontend-dev-guide/theme-best-practice.md (hunk 2 above)
Comment (shrielenee) [reply guides/v2.2/frontend-dev-guide/theme-best-practice.md]: Update and upgrade are not always the same thing, and definitely not always the same for Magento. You can upgrade from Open Source to Commerce, but you can't update from there to there. You can update your installation with hotfixes, but that wouldn't be considered an upgrade.

Comment (hickskenh) [reply guides/v2.2/frontend-dev-guide/theme-best-practice.md]: valid.  wasn't thinking from edition to edition, was thinking version updates.

**Comment by shrielenee** [comment]
running tests
